src/main/scala/com/redislabs/provider/redis/util/SparkUtils.scala

```diff
@@ -0,0 +1,17 @@
+package com.redislabs.provider.redis.util
+
+import org.apache.spark.sql.types.StructType
+
+object SparkUtils {
+  /**
+   * Setting the schema column positions the same order as in requiredFields
+   * @param schema Current schema
+   * @param requiredColumns Column positions expecting by Catalyst
+   */
+  def alignSchemaWithCatalyst(schema: StructType, requiredColumns: Seq[String]): StructType = {
```

Comment:
Looks like these changes are not related to Sentinels support. Could you please create a separate PR for that?

Reply:
Added another PR https://github.com/RedisLabs/spark-redis/pull/252